package.json

```diff
@@ -19,6 +19,7 @@
   },
```

Comment:
Why is the package.json file outside the application.

Reply:
This has to do with how I am pushing changes to the git repo....
I will experiment and solve this.